easybuild/FastQC-0.11.8-Java-1.8.0_181.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PackedBinary'
+
+name = 'FastQC'
+version = '0.11.8'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'
+description = """FastQC is a quality control application for high throughput
+sequence data. It reads in sequence data in a variety of formats and can either
+provide an interactive application to review the results of several different
+QC checks, or create an HTML based report which can be integrated into a
+pipeline."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
+sources = ['%(namelower)s_v%(version)s.zip']
+patches = ['FastQC_shebang.patch']
+
+dependencies = [('Java', '1.8.0_181')]
```

Comment:
@fizwit Here too, we should start using the `Java/1.8` wrapper rather than specifying a specific version of `Java`, cfr. https://easybuild.readthedocs.io/en/develop/Wrapping_dependencies.html

Reply:
I removed FastQC from this PR.

Reply:
Hmm, no, it's actually still here... Seems like you added c45c9bd to to wrong branch?

Shouldn't be hard to fix: `git checkout Bio-DEF; git cherry-pick c45c9bd; git push ... Bio-DEF` where you replace the `...` with the name of your remote

Reply:
ops, moved FastQC back to this PR and pushed.